### ISSUE TITLE
Force Window's Width

### ISSUE THREAD

**Issue by Alx06** [Microsoft/WinObjC#244]
I've a Portrait app.
But the window used to display it is a square.
Can I force the window's size?
Can I force the window to fit my app screen size?

**Comment by Alx06** [comment]
If I can't, is there a way to change the backgrpund color of the window at left and right of my app? Can I provide a background image for that?

**Comment by davelamb** [comment]
The lack of resizing was addressed in 10.0.10240 when they added the ApplicationView::TryResizeView API, but trying it locally it's not behaving as expected. The official story is your app should be able to run at any size above some minimum (which you can specify programmatically with ApplicationView::SetPreferredMinSize) but obviously that's not always possible.

The size of the UIWindow compared to the app view can be controlled with some WinObjC specific code, as seen in WOCCatalog, AppDelegate.m right at the top. When you change the window size with no fixed width or height and autoMagnification/sizeUIWindowToFit enabled, you'll be notified of the change through the standard layout path. This introduces a little more work, since you'd have to modify your view controllers to return a fixed aspect view, and add some background UIImageViews and whatnot, but at the moment we don't have any other way modify the letterboxes.

Keep in mind we do have projections to WinRT, so there might be another path you can exercise to sneak in some customization, but that'd be a stop-gap solution at best.

tl;dr: You can fit your app to your screen size through the UIApplicationInitialStartupMode category on UIApplication. We don't yet have customization for letterboxing, but it's a good idea! We'll get to this next chance we get. 

Thanks!
